src/nmea.c

```diff
@@ -54,7 +55,8 @@ static struct nmea_dispatcher *nmea_dispatchers_head;
  * max_len = max possible length of the body of the message
  * (not including suffix)
  */
-#define NMEA_SENTENCE_START(max_len) char sentence_buf[max_len + NMEA_SUFFIX_LEN]; \
+#define NMEA_SENTENCE_START(max_len) \
```

Comment:
It wouldn't work, anyway. The allocated variables need to be visible to the other macros.